README.md

```diff
@@ -1,106 +1,170 @@
+   repository by running the following but replacing `$VERSION` with the appropriate version
+   of the API:
+   ```sh-session
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION.jar
+
+   $ # or
+
+   $ mvn org.apache.maven.plugins:maven-install-plugin:2.5.2:install-file \
+       -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar
+
+   $ or
+   $ mvn install:install-file -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar \
+       -DgroupId=net.conjur.api \
+       -DartifactId=conjur-api \
+       -Dversion=$VERSION \
+       -Dpackaging=jar
+   ```
 
 ## Configuration
-Once the setup steps have been successfully run, we will now define the environment variables needed to make the connection between the new app and Conjur. The best way to do so is by setting environment variables. 
-In Conjur (both OSS and DAP), environment variables are mapped to configuration variables by prepending `CONJUR_` to the all-caps name of the
-configuration variable. For example, `appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.  
 
-_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` environment variables ONLY) in the app instead of environment variables, see the [Authorization Examples](#authorization-examples) section below.
+Once the setup steps have been successfully run, we will now define the variables needed
+to make the connection between the new app and Conjur. You can do this by setting
+[environment variables](#environment-variables), [system properties](#system-properties),
+or some combination of both.
 
-The following environment variables need to be included in the apps runtime environment in order use the Conjur API:
-- `CONJUR_ACCOUNT`
-- `CONJUR_AUTHN_LOGIN`
-- `CONJUR_AUTHN_API_KEY`
-- `CONJUR_APPLIANCE_URL`
-- `CONJUR_AUTHN_URL`
+_NOTE:_ System properties will override enviroment values when both are defined for a
+variable.
 
-`CONJUR_ACCOUNT` - account specified during Conjur setup
+### Environment Variables
 
-`CONJUR_APPLIANCE_URL` - Conjur HTTPS endpoint (OSS/DAP)
+In Conjur (both OSS and DAP), environment variables are mapped to configuration variables
+by prepending `CONJUR_` to the all-caps name of the configuration variable. For example,
+`appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.
 
-`CONJUR_AUTHN_LOGIN` - user/host identity
+_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`
+environment variables ONLY) in the app instead of environment variables, see the
+[Authorization Examples](#authorization-examples) and/or [system properties](#system-properties)
+section below.
 
-`CONJUR_AUTHN_API_KEY` - user/host API key
+The following environment variables need to be included in the apps runtime environment in
```

Comment:
```suggestion
The following environment variables need to be included in the app's runtime environment in
```

Reply:
This is intentional. They don't _have_ to be included - they can be supplemented by system properties or inlined code

Reply:
ok, updated with a qualifier and still want to get that apostrophe in :)